scripts/buildcommands.py

```diff
@@ -42,7 +43,7 @@ def update_data_dictionary(self, data):
         pass
     def generate_code(self, options):
         code = []
-        for funcname, funcs in self.call_lists.items():
+        for funcname, funcs in list(self.call_lists.items()):
```

Comment:
In python2 `dict.items()` returns a list, in python3 it returns an generator.
So you added `list()` to convert the iterable generator to a list, and then iterate over it.

This seems completely unnecessary.
Please only convert to list where it's really required, else you're just introducing unnecessary complexity and a performance hit.